### PR TITLE
Renames Kabobs to Kebabs. Fixes Kebab-Related Issues. Expands Kebab Production Methods.

### LINKED ISSUE #25224: Meatwheat Clumps Cannot be Used to Make Meat-Kabobs
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1633

### Issue Description

2 meat and a metal rod make a meat-kabob. Meatwheat is supposed to be a botany-based meat-substitute. You should be able to use it, yeah?

### What did you expect to happen?

For my 2 meatwheat and metal rod to become tasty meat-kabob.

### What happened instead?

It became a burned mess instead :(

### Why is this bad/What are the consequences?

Cannot make meat-kabob using GMO plant meat. :(

### Steps to reproduce the issue.

Put 1 metal rod and 2 meatwheat clumps into a grill, activate.

### When did the problem start happening?

No clue, probably forever ago.

### Extra information

_No response_

### Relevant log output/runtime error

_No response_

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Resolves #25224. Regular meat and meat-substitutes can now be used to make meat-kebabs, not just monkey meat.

Synthimeat now creates synthimeat-kebabs instead of normal meat-kebabs.

## Why It's Good For The Game
Arbitrary limitations on what kind of meat a kebab can be made from is bad.

Synthimeat is now represented in the kebab industry.

Kabob is less commonly used than kebab, even in the USA where the spelling originates.

## Testing
Cooked kebabs out of all the different meat types. They cooked properly into the expected types of kebab.

## Changelog
:cl:
add: Adds synthimeat-kebabs.
tweak: Kabobs are now kebabs. Allows all meat types to be used in the creation of meat-kebabs.
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/recipes/recipes_grill.dm [hunk 1]
```diff
@@ -117,39 +117,39 @@
...
+	result = /obj/item/food/snacks/syntikebab
 
-/datum/recipe/grill/syntikabob
+/datum/recipe/grill/meatkebab	// Do not put this above the other meat kebab variants or it will override them.
 	items = list(
 		/obj/item/stack/rods,
-		/obj/item/food/snacks/meat/syntiflesh,
-		/obj/item/food/snacks/meat/syntiflesh,
+		/obj/item/food/snacks/meat/,
+		/obj/item/food/snacks/meat/
```
Comment (Henri215) [inline code/modules/food_and_drinks/recipes/recipes_grill.dm:140]:
```suggestion
		/obj/item/food/snacks/meat,
		/obj/item/food/snacks/meat
```

**Comment by DGamerL** [comment]
Imo, kebab being only producible from humanoids is pretty funny

**Comment by Fordoxia** [comment]
> Imo, kebab being only producible from humanoids is pretty funny

It makes no sense, though.

**Comment by Oyu07** [comment]
image

Design team, let this man cook

File: code/modules/food_and_drinks/recipes/recipes_grill.dm [hunk 2]
```diff
@@ -124,39 +124,38 @@
 	)
 	result = /obj/item/food/snacks/wingfangchu
 
-/datum/recipe/grill/human/kabob
+/datum/recipe/grill/human/kebab
```
Comment (FunnyMan3595) [inline code/modules/food_and_drinks/recipes/recipes_grill.dm:127]: I'm pretty sure this needs a proc override to rename the resulting kebab to include the name(s) of who it was made from.

**Review by FunnyMan3595 — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/recipes/recipes_grill.dm (hunk 2 above)
Comment (Fordoxia) [reply code/modules/food_and_drinks/recipes/recipes_grill.dm:127]: It's a subtype of an arbitrary human food found in sandwiches.dm, which also handles human burgers.

**Review by FunnyMan3595 — APPROVED** [review]

File: code/modules/food_and_drinks/food/foods/ethnic.dm [hunk 3]
```diff
@@ -135,35 +135,43 @@
...
+	icon_state = "kebab"
+	desc = "Delicious synthetic meat, on a stick."
+	trash = /obj/item/stack/rods
+	filling_color = "#A85340"
+	list_reagents = list("nutriment" = 8)
+
+/obj/item/food/snacks/human/kebab
+	name = "-kebab"
+	icon_state = "kebab"
+	desc = "Human meat, on a stick."
```
Comment (Henri215) [inline code/modules/food_and_drinks/food/foods/ethnic.dm:158]:
```suggestion
/obj/item/food/snacks/human/kebab
	name = "-kebab"
	desc = "Human meat, on a stick."
	icon_state = "kebab"
```
Its written "-kebab" to make it look different from the customizable version?
image

File: code/modules/food_and_drinks/food/foods/ethnic.dm [hunk 4]
```diff
@@ -135,35 +135,43 @@
...
+	goal_difficulty = FOOD_GOAL_NORMAL
 
-/obj/item/food/snacks/monkeykabob
-	name = "meat-kabob"
-	icon_state = "kabob"
-	desc = "Delicious meat, on a stick."
+/obj/item/food/snacks/syntikebab
+	name = "synthimeat-kebab"
+	icon_state = "kebab"
+	desc = "Delicious synthetic meat, on a stick."
```
Comment (Henri215) [inline code/modules/food_and_drinks/food/foods/ethnic.dm:150]:
```suggestion
	name = "synthimeat-kebab"
	desc = "Delicious synthetic meat, on a stick."
	icon_state = "kebab"
```

File: code/modules/food_and_drinks/food/foods/ethnic.dm [hunk 5]
```diff
@@ -135,35 +135,43 @@
...
 //////////////////////
 
-/obj/item/food/snacks/human/kabob
-	name = "-kabob"
-	icon_state = "kabob"
-	desc = "Human meat, on a stick."
+/obj/item/food/snacks/meatkebab
+	name = "meat-kebab"
+	icon_state = "kebab"
+	desc = "Delicious meat, on a stick."
```
Comment (Henri215) [inline code/modules/food_and_drinks/food/foods/ethnic.dm:141]:
```suggestion
	name = "meat-kebab"
	desc = "Delicious meat, on a stick."
	icon_state = "kebab"
```

File: code/modules/food_and_drinks/food/foods/ethnic.dm [hunk 6]
```diff
@@ -135,35 +135,43 @@
...
 	list_reagents = list("nutriment" = 8)
-	goal_difficulty = FOOD_GOAL_NORMAL
 
-/obj/item/food/snacks/tofukabob
-	name = "tofu-kabob"
-	icon_state = "kabob"
+/obj/item/food/snacks/tofukebab
+	name = "tofu-kebab"
+	icon_state = "kebab"
 	desc = "Vegan meat, on a stick."
```
Comment (Henri215) [inline code/modules/food_and_drinks/food/foods/ethnic.dm:166]:
```suggestion
	name = "tofu-kebab"
	desc = "Vegan meat, on a stick."
	icon_state = "kebab"
```

File: code/modules/food_and_drinks/food/foods/ethnic.dm (hunk 3 above)
Comment (Fordoxia) [reply code/modules/food_and_drinks/food/foods/ethnic.dm:158]: snacks/human should take the name of player meat and add it in front of the hyphen (e.g. Goobina-kebab).

**Review by Henri215 — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**